src/main/java/com/uwetrottmann/tmdb2/TmdbInterceptor.java

```diff
@@ -79,7 +79,7 @@ public static Response handleIntercept(Chain chain, Tmdb tmdb) throws IOExceptio
                 try {
                     Integer retry = Integer.parseInt(retryAfter);
                     Thread.sleep((int) ((retry + 0.5) * 1000));
-                    response = chain.proceed(builder.build());
+                    return handleIntercept(chain,tmdb);
                 } catch (Exception exc) {
```

Comment:
Just because I'm seeing it right now: if you do not intend on catching an exception, please name it `ignored` instead of `exc`. Most IDEs will pick up on that and not warn you about an empty catch block.